docs/ionic/authentication.md

```diff
@@ -0,0 +1,94 @@
+# STEP 1: install this plugin: 
+```
+1) ionic cordova plugin add cordova-universal-links-plugin
+2) ionic cordova plugin add cordova-plugin-buildinfo
+3) ionic cordova plugin add cordova-plugin-browsertab
+4) ionic cordova plugin add cordova-plugin-inappbrowser
+5) ionic cordova plugin add cordova-plugin-customurlscheme (for ios)
+```
+
+# STEP 2: Add Firebase to your Ionic
+
+  STEP 2.1:  [setup firebase to angular project] 
+  (https://github.com/angular/angularfire2/blob/master/docs/install-and-setup.md)
+
+  STEP 2.2: To set up an Android app, go to [Firebase Console](https://console.firebase.google.com/) then 
+  Click **Add Firebase to your Android app** and follow the setup steps.
+
+# STEP 3: Set up Firebase Authentication for Cordova ( sumary from [firebase instruction](https://firebase.google.com/docs/auth/web/cordova))
+
+ STEP 3.1: In the Firebase console, open the **Dynamic Links** section at bottom left panel, setup by they instruction
+
+ STEP 3.2: add this to config.xml at root level of project:
```

Comment:
Use bold, capitalize "add", and backticks for `config.xml`

**Add this to `config.xml` at root level of project*